resources/backup.yaml

```diff
@@ -1,3 +1,45 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: stolon-backup
+---
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+kind: Role
+metadata:
+  name: stolon-backup
+  namespace: default
+rules:
+  - apiGroups: [ "" ]
+    verbs:     [ "get" , "list" , "watch" ]
+    resources:
+      - secrets
+    resourceNames:
+      - "stolon"
```

Comment:
Style nitpick: this looks out of place with some literals unquoted and alternating lists styles: [lists] vs -lists